solutions/requirements.txt

```diff
@@ -0,0 +1,330 @@
+# This file may be used to create an environment using:
```

Comment:
please, replace this file with requirements.txt containing these libraries: scrapy, protego, wordcloud, pillow, beautifulsuop4